cl/search/feeds.py

```diff
@@ -97,6 +97,38 @@ def item_title(self, item):
     def item_categories(self, item):
         return [item['status'], ]
 
+    def item_enclosure_url(self, item):
+        try:
+            path = item['local_path']
+            if not path.startswith('/'):
+                return '/%s' % (path,)
+            return path
+        except:
+            return None
+
+    def item_enclosure_length(self, item):
+        try:
+            file_loc = os.path.join(
+                settings.MEDIA_ROOT,
+                item['local_path'].encode('utf-8')
+            )
+            return os.path.getsize(file_loc)
+        except:
+            return None
+
+    def item_enclosure_mime_type(self, item):
+        try:
+            file_loc = os.path.join(
+                settings.MEDIA_ROOT,
+                item['local_path'].encode('utf-8')
+            )
+            mime = magic.from_file(file_loc, mime=True)
+            if mime.startswith('inode'):
+                return None
+            return mime
```

Comment:
The parent directory will be the same as the extension. They're set the same way, so if there's no extension at all, feel free to go straight to application/octet-stream.